spec/game_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "./lib/game"
+require "./lib/player"
+require "./lib/module"
+
+RSpec.describe Game do
+
+    let(:player1) {Player.new("player1", "X")}
+    let(:player2) {Player.new("player2", "O")}
+    let(:board) {Board.new}
+    let(:piece) { "x" }
+    let(:piece2) { "o" }
+
+    describe "#check_draw" do
+
+        it "returns true and a message in case of draw and false in the opposite" do
+            allow(board).to receive(:full?).and_return(true)
+            expect(board.full?).to eq(true)
```

Comment:
You're testing the **Game** class, not the **Board** one. 
So I think you should rather have
`expect(game.check_draw).to eq(true)`